src/MiNET/MiNET/Blocks/Stationary.cs

```diff
@@ -30,14 +30,23 @@
 
 namespace MiNET.Blocks
 {
-	public class Stationary : Block
+	public interface IStationary
+	{
+		int LiquidDepth { get; set; }
+	}
```

Comment:
Is this used? As the stationary class doesn't implement it?

Reply:
done